### ISSUE TITLE
Support for docblocks / autocomplete?

### ISSUE THREAD

**Issue by tspence** [dotnet/vscode-csharp#4490]
Hi all - I'm new to this language server but am curious whether it supports autocomplete for C# docblocks.  

Some of my API projects make use of embedded C# docblocks and annotations to export documentation for our developer site.  In the past, I would type `///` into visual studio and it would pre-fill a docblock for my API method; I then had an analyzer that would examine the file, extract the code comments, and publish them to the developer site.

Is this something that's supported?  I can't find any reference to docblocks in the settings.

I didn't see any existing issues mentioning this item, so perhaps this is an intentional design decision?

## Steps to reproduce
In front of a public class member or method, type `///` to begin a docblock.

## Expected  behavior
The docblock is auto-filled out using the template:

```
/// <summary>
/// (cursor placed here)
/// </summary>
/// <param name="p1"></param>
```

## Actual behavior

Nothing happens

**Comment by JoeRobich** [comment]
@tspence We support this through the format on type option. If you have format on type enabled but are not seeing this behavior, please let me know. Do note that the cursor won't be placed where you expect at this time.

![image](https://user-images.githubusercontent.com/611219/114895788-a1e19b00-9dc4-11eb-8bf1-271fc0ca7241.png)

**Comment by tspence** [comment]
I just enabled this feature, then restarted VS Code and typed the following in a CS file:

```
public class X {
}
```

I put my cursor above the class definition and typed `///`.  Nothing happened.

I then tried adding a method within the class like this:

```
 public class X {
    ///
    public void GetFunctionName() {

    }
}
```

Again, typing the slash-slash-slash above the function name didn't produce any docblocks.  

That said, I appear to be using `ms-dotnettools.csharp` version 1.23.11, which claims it is powered by OmniSharp.  Is it possible I'm using the wrong extension here?
